overlord/devicestate/firstboot_test.go

```diff
@@ -1160,14 +1160,12 @@ type: kernel`
 		kernelFname = fname
 	}
 
-	if !opts.classic {
```

Comment:
this change looks strange, why do we need volumes for classic now?

Reply:
Reverted along with the classic-gadget in seed tests. This was left behind from the initial version of the code.